Gemfile

```diff
@@ -21,16 +21,26 @@ gem 'turbolinks'
 # Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
 gem 'jbuilder', '~> 2.0'
 # bundle exec rake doc:rails generates the API under doc/api.
-gem 'sdoc', '~> 0.4.0',          group: :doc
+gem 'sdoc', '~> 0.4.0', group: :doc
 
-# Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
-gem 'spring',        group: :development
+# design
+gem "font-awesome-rails"
+gem 'bourbon'
+gem 'neat'
+gem "normalize-rails"
 
-#for user authentication
-gem 'devise'
+group :development, :test do
```

Comment:
Do we really want to limit these gems to development and test environments? Won't we need Devise in production?